plugins/modules/dconf.py

```diff
@@ -390,6 +407,64 @@ def reset(self, key):
         # Value was changed.
         return True
 
+    def load(self, key, path):
+        """
+        Load the config file in specified path.
+
+        if an error occurs, a call will be made to AnsibleModule.fail_json.
+
+        :param key: dconf directory for which the config should be set.
+        :type key: str
+
+        :param path: Remote configuration path to set for the specified dconf path.
+        :type value: str
+
+        :returns: bool -- True if a change was made, False if no change was required.
+        """
+        # Ensure key refers to a directory, as required by dconf
+        root_dir = key
+        if not root_dir.endswith('/'):
+            root_dir += '/'
+
+        # Read config to check if change is needed and passing to command line
+        try:
+            with open(path, 'r') as fd:
+                raw_config = fd.read()
+        except FileNotFoundError as ex:
+            self.module.fail_json(msg='dconf failed while reading configuration file with error: %s' % ex)
```

Comment:
dconf wasn't involved yet:
```suggestion
            self.module.fail_json(msg='Failed while reading configuration file %s with error: %s' % (path, ex))
```